model_templates/gpu_nemo_tensorrt_llm_textgen/README.md

```diff
@@ -0,0 +1,65 @@
+## NeMo Inference Server Text Generation Template
+
+This text generation example uses Llama-2-7b model pre-compiled for A100 GPU.
+
+## Requirements
+- NeMo Inference Models currently only available via Early Access program:
```

Comment:
I don't think this is true.

Reply:
There are a lot of other changes in this PR so probably good to get it merged mostly as-is and update early-access stuff once we get access ourselves.